src/bindings/js/node/lib/addon.ts

```diff
@@ -299,6 +299,39 @@ interface Model {
    * @param name The string to set as the friendly name.
    */
   setFriendlyName(name: string): void;
+  /** Reshapes model input.
+   * @param partialShape The {@link PartialShape} object
+   * to reshape the model input.
+   * @param variablesShapes New shapes for variables.
+   */
+  reshape(
+    partialShape: PartialShape,
+    variablesShapes: Record<string, PartialShape | string>
```

Comment:
Is the `variablesShapes` parameter required? As I can see `ModelWrap::reshape` can process without it.

Reply:
It's optional. I added more tests :)